test/validation/test_formatting_validation.py

```diff
@@ -176,6 +177,22 @@ def test_check_for_fenced_code_block(test_input: str, should_match: bool) -> Non
     assert actual == expected
 
 
+@pytest.mark.parametrize(
+    "test_input,should_match",
+    [
+        ("—-", True),
+        ("-—", True),
+        ("---------", False),
+        ("Word\n    int x = 1\nWord", False),

```

Comment:
I would add `("---", False)` here as well, as this is the most commonly used separator string.